firebase-firestore/src/main/java/com/google/firebase/firestore/core/Query.java

```diff
@@ -57,11 +57,17 @@ public static Query atPath(ResourcePath path) {
 
   private final List<OrderBy> explicitSortOrder;
 
-  private List<OrderBy> memoizedOrderBy;
+  private List<OrderBy> memoizedNormalizedOrderBys;
 
-  // The corresponding Target of this Query instance.
+  /** Returns a `Target` instance this query will be mapped to in backend and local store. */
   private @Nullable Target memoizedTarget;
 
+  /**
+   * Returns a `Target` instance this query will be mapped to in backend and local store, for use
+   * within an aggregate query.
```

Comment:
During Web code review, there was feedback to add additional comments describing the difference between aggregate and non-aggregate target. Consider adding [this additional info](https://github.com/firebase/firebase-js-sdk/pull/7507/files#diff-88195d5b635ef88c28a2fe6e379f36fdbbe7059dab7b92e49940da39a6335c61R77) to the comment.